cmdk/package.json

```diff
@@ -26,9 +26,9 @@
     "react-dom": "^18.0.0"
   },
   "dependencies": {
-    "@radix-ui/react-dialog": "1.0.5",
+    "@radix-ui/react-dialog": "^1.1.1",
     "@radix-ui/react-id": "^1.0.1",
-    "@radix-ui/react-primitive": "1.0.3"
+    "@radix-ui/react-primitive": "^2.0.0"
```

Comment:
Couldn't find breaking changes in https://www.radix-ui.com/primitives/docs/overview/releases so I opened https://github.com/radix-ui/primitives/issues/2976